build/lib/locFunc.js

```diff
@@ -0,0 +1,162 @@
+"use strict";
```

Comment:
Please add xml doc comments to these functions explaining what they're doing.

Reply:
Done so